### ISSUE TITLE
Build fails on Linux: `FE_*` redefined here

### ISSUE THREAD

**Issue by CounterPillow** [Limeth/obs-shaderfilter-plus#39]
```
error[E0428]: the name `FE_INVALID` is defined multiple times
    --> /home/fratti/Projekte/obs-shaderfilter-plus/target/release/build/obs-sys-0941a661be0ede12/out/bindings.rs:7844:1
     |
499  | pub const FE_INVALID: u32 = 1;
     | ------------------------------ previous definition of the value `FE_INVALID` here
...
7844 | pub const FE_INVALID: _bindgen_ty_3 = 1;
     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `FE_INVALID` redefined here
     |
     = note: `FE_INVALID` must be defined only once in the value namespace of this module

error[E0428]: the name `FE_DIVBYZERO` is defined multiple times
    --> /home/fratti/Projekte/obs-shaderfilter-plus/target/release/build/obs-sys-0941a661be0ede12/out/bindings.rs:7846:1
     |
500  | pub const FE_DIVBYZERO: u32 = 4;
     | -------------------------------- previous definition of the value `FE_DIVBYZERO` here
...
7846 | pub const FE_DIVBYZERO: _bindgen_ty_3 = 4;
     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `FE_DIVBYZERO` redefined here
     |
     = note: `FE_DIVBYZERO` must be defined only once in the value namespace of this module

error[E0428]: the name `FE_OVERFLOW` is defined multiple times
    --> /home/fratti/Projekte/obs-shaderfilter-plus/target/release/build/obs-sys-0941a661be0ede12/out/bindings.rs:7847:1
     |
501  | pub const FE_OVERFLOW: u32 = 8;
     | ------------------------------- previous definition of the value `FE_OVERFLOW` here
...
7847 | pub const FE_OVERFLOW: _bindgen_ty_3 = 8;
     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `FE_OVERFLOW` redefined here
     |
     = note: `FE_OVERFLOW` must be defined only once in the value namespace of this module

error[E0428]: the name `FE_UNDERFLOW` is defined multiple times
    --> /home/fratti/Projekte/obs-shaderfilter-plus/target/release/build/obs-sys-0941a661be0ede12/out/bindings.rs:7848:1
     |
502  | pub const FE_UNDERFLOW: u32 = 16;
     | --------------------------------- previous definition of the value `FE_UNDERFLOW` here
...
7848 | pub const FE_UNDERFLOW: _bindgen_ty_3 = 16;
     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `FE_UNDERFLOW` redefined here
     |
     = note: `FE_UNDERFLOW` must be defined only once in the value namespace of this module

error[E0428]: the name `FE_INEXACT` is defined multiple times
    --> /home/fratti/Projekte/obs-shaderfilter-plus/target/release/build/obs-sys-0941a661be0ede12/out/bindings.rs:7849:1
     |
503  | pub const FE_INEXACT: u32 = 32;
     | ------------------------------- previous definition of the value `FE_INEXACT` here
...
7849 | pub const FE_INEXACT: _bindgen_ty_3 = 32;
     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `FE_INEXACT` redefined here
     |
     = note: `FE_INEXACT` must be defined only once in the value namespace of this module

error[E0428]: the name `FE_TONEAREST` is defined multiple times
    --> /home/fratti/Projekte/obs-shaderfilter-plus/target/release/build/obs-sys-0941a661be0ede12/out/bindings.rs:7851:1
     |
505  | pub const FE_TONEAREST: u32 = 0;
     | -------------------------------- previous definition of the value `FE_TONEAREST` here
...
7851 | pub const FE_TONEAREST: _bindgen_ty_4 = 0;
     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `FE_TONEAREST` redefined here
     |
     = note: `FE_TONEAREST` must be defined only once in the value namespace of this module

error[E0428]: the name `FE_DOWNWARD` is defined multiple times
    --> /home/fratti/Projekte/obs-shaderfilter-plus/target/release/build/obs-sys-0941a661be0ede12/out/bindings.rs:7852:1
     |
506  | pub const FE_DOWNWARD: u32 = 1024;
     | ---------------------------------- previous definition of the value `FE_DOWNWARD` here
...
7852 | pub const FE_DOWNWARD: _bindgen_ty_4 = 1024;
     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `FE_DOWNWARD` redefined here
     |
     = note: `FE_DOWNWARD` must be defined only once in the value namespace of this module

error[E0428]: the name `FE_UPWARD` is defined multiple times
    --> /home/fratti/Projekte/obs-shaderfilter-plus/target/release/build/obs-sys-0941a661be0ede12/out/bindings.rs:7853:1
     |
507  | pub const FE_UPWARD: u32 = 2048;
     | -------------------------------- previous definition of the value `FE_UPWARD` here
...
7853 | pub const FE_UPWARD: _bindgen_ty_4 = 2048;
     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `FE_UPWARD` redefined here
     |
     = note: `FE_UPWARD` must be defined only once in the value namespace of this module

error[E0428]: the name `FE_TOWARDZERO` is defined multiple times
    --> /home/fratti/Projekte/obs-shaderfilter-plus/target/release/build/obs-sys-0941a661be0ede12/out/bindings.rs:7854:1
     |
508  | pub const FE_TOWARDZERO: u32 = 3072;
     | ------------------------------------ previous definition of the value `FE_TOWARDZERO` here
...
7854 | pub const FE_TOWARDZERO: _bindgen_ty_4 = 3072;
     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `FE_TOWARDZERO` redefined here
     |
     = note: `FE_TOWARDZERO` must be defined only once in the value namespace of this module

For more information about this error, try `rustc --explain E0428`.
error: could not compile `obs-sys` due to 9 previous errors
```
Rust version:
```
$ rustc --version
rustc 1.65.0 (Arch Linux rust 1:1.65.0-1)
```

**Comment by sikha-root** [comment]
Duplicate of https://github.com/Limeth/rust-obs-plugins/issues/1, which hasn't been triaged either....
